kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -37,6 +37,11 @@ public class ResponsiveConfig extends AbstractConfig {
 
   // ------------------ connection configurations -----------------------------
 
+  public static final String COMPATIBILITY_MODE_CONFIG = "responsive.compatibility.mode";
+  private static final String COMPATIBILITY_MODE_DOC = "This configuration enables running Responsive "
+      + "in compatibility mode, disabling certain features.";
```

Comment:
compatibility is a strange word for this. Maybe "instrumentation"?

Reply:
the idea was whether it should be compatible with existing stream deployments or not - basically I wanted to communicate that if you run it in "compatibility mode" you will not be able to use responsive stores.